src/business-logic/services/inews-gateway-connection-implementation.ts

```diff
@@ -88,36 +160,52 @@ export class INewsGatewayDeviceConnection implements DeviceConnection {
 
     this.pingTimeout = setTimeout(() => {
       this.terminate()
-    }, 30000 + 1000)
+    }, 300_000)
```

Comment:
I want to have different devices store their timeout / reconnect values with the device. Does that make sense? i.e. Device.Timeout = 300_000, or, to follow your logic: 

const INEWS_TIMEOUT_VALUE: number = 300_000
iNewsDevice.Timeout = device.timeout ?? INEWS_TIMEOUT_VALUE

Reply:
With different devices do mean that an iNews and Atem device have different timeouts or that you could configure two iNews devices that have different timeouts?

If it's the first, then the timeout is already device specific because you are in `INewsGatewayConnectionImplementation` and then I would do:
```
const TIMEOUT_DURATION: number = 300_000

export class INewsGatewayConnectionImplementation {
  ...

  private heartbeat(): void {
    this.pingTimeout = setTimeout(() => {...}, TIMEOUT_DURATION)
  }
}
```

If it's the latter, then I ask: Is that something we need?